src/tests/integration/test-data/foundry/test/PlainPrankTest.t.sol

```diff
@@ -97,6 +97,15 @@ contract PlainPrankTest is Test {
         vm.stopPrank();
     }
 
+    function test_startPrank_consecutive() public {
+        AdditionalToken token = new AdditionalToken();
+        vm.startPrank(address(0));
+        token.incrementCount();
+        vm.startPrank(address(1));
+        vm.stopPrank();
+        assert(token.count() == 1);
```

Comment:
Could we have an assertion about the pranking address here, just to see it's working properly?